src/main/interface/msp.c

```diff
@@ -1330,9 +1330,9 @@ bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, currentPidProfile->dFilter[ROLL].Wc);
         sbufWriteU8(dst, currentPidProfile->dFilter[PITCH].Wc);
         sbufWriteU8(dst, currentPidProfile->dFilter[YAW].Wc);
-        sbufWriteU8(dst, gyroConfig()->averagedGyro[ROLL]);
-        sbufWriteU8(dst, gyroConfig()->averagedGyro[PITCH]);
-        sbufWriteU8(dst, gyroConfig()->averagedGyro[YAW]);
+        sbufWriteU8(dst, 0);
```

Comment:
either i dont understand how the buffering works or one statement of the same should be enough